oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java

```diff
@@ -381,6 +431,26 @@ public Configuration build() throws InternalException {
                 servletRoutingPathSuffixAssertion = "assertionConsumer";
             }
 
+            if (auditLoggerClassName == null || auditLoggerClassName.length() == 0) {
```

Comment:
Don't know if we can use Apache StringUtils, but for all these checks `isBlank()` would, IMO, be a less noisy and erro prone approach.